mne/bem.py

```diff
@@ -1063,14 +1063,20 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
     run_subprocess(cmd, env=env, stdout=sys.stdout, stderr=sys.stderr)
 
     if op.isfile(T1_mgz):
-        # XXX : do this with python code
+        new_info = _extract_volume_info(T1_mgz)
+        if new_info is None:
+            warn('nibabel is required to replace the volume info. Volume info'
```

Comment:
Not sure what you mean -- I just mean that the string should say what version people need. Because it doesn't have that currently, people with the current or an older release of nibabel will be confused by this warning